src/shared/codelens/codeLensUtils.ts

```diff
@@ -352,3 +327,35 @@ export function initializeTypescriptCodelens(context: ExtContext): void {
         })
     )
 }
+
+export async function findParentProjectFile(
```

Comment:
this could probably go in `workspaceUtils` or something like that. not codelens related.

Reply:
Will do, this is also creating a circular dependency